tests/shared/helpers/notifyServer.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from "@playwright/test"
+
+export class NotifyServer {
+  private notifyBaseUrl = "https://notify.walletconnect.com"
+
+  public async sendMessage({
+    projectId,
+    projectSecret,
+    accounts,
+    url,
+    title,
+    body,
+    icon,
+    type
+  }: {
+    projectId: string,
+    projectSecret: string,
+    accounts: string[]
+    title: string,
+    body: string,
+    icon: string,
+    url: string
+    type: string
+  }) {
+    const request = JSON.stringify({
+      accounts,
+      notification: {
+	title,
+	body,
+	icon,
+	url,
+	type
+      }
+    })
+
+    const fetchUrl = `${this.notifyBaseUrl}/${projectId}/notify`
+
+    const headers = new Headers({
+      Authorization: `Bearer ${projectSecret}`,
+      "Content-Type": "application/json"
+    })
+
+    const fetchResults = await fetch(fetchUrl, {
+      method: "POST",
+      headers,
+      body: request
+    })
+
+    console.log({fetchResultsStatus: fetchResults.status, fetchResults: await fetchResults.text()})
```

Comment:
Need to keep log?

Reply:
Yes I think it would be a quick way to know what went wrong when running the tests locally or if you want to inspect why a CI pipeline failed